src/cassy.py

```diff
@@ -557,3 +557,553 @@ def change_user_email(username, new_email):
+
+    try:
+        password = get_user_password(username)
+        if not password:
+            raise PlantalyticsAuthException(USER_INVALID)
+        parameters['password'] = password
+        prepared_statement = session.prepare(
+            query.format(table)
+        )
+        session.execute(
+            prepared_statement,
+            parameters
+        )
+        return True
+    # Known exception
+    except PlantalyticsLoginException as e:
+            raise PlantalyticsAuthException(USER_INVALID)
+    except PlantalyticsException as e:
+        raise e
+    # Unknown exception
+    except Exception as e:
+        raise Exception('Transaction Error Occurred: '.format(str(e)))
+
+
+def disable_user(username):
+    """
+    Disables user in DB for submitted username.
+    """
+
+    session.row_factory = named_tuple_factory
+    table = str(os.environ.get('DB_USER_TABLE'))
+    parameters = {
+        'username': username,
+        'enable': False,
+    }
+    query = (
+        'UPDATE {} SET enable=? WHERE username=? AND password=?;'
+    )
+
+    try:
+        password = get_user_password(username)
+        if not password:
+            raise PlantalyticsAuthException(USER_INVALID)
+        parameters['password'] = password
+        prepared_statement = session.prepare(
+            query.format(table)
+        )
+        session.execute(
+            prepared_statement,
+            parameters
+        )
+        return True
+    # Known exception
+    except PlantalyticsLoginException as e:
+            raise PlantalyticsAuthException(USER_INVALID)
+    except PlantalyticsException as e:
+        raise e
+    # Unknown exception
+    except Exception as e:
+        raise Exception('Transaction Error Occurred: '.format(str(e)))
+
+
+def edit_user(user_edit_info):
+    """
+    Edits user info in DB using submitted info.
+    """
+
+    session.row_factory = named_tuple_factory
+    table = str(os.environ.get('DB_USER_TABLE'))
+    parameters = {
+        'username': user_edit_info.get('username', ''),
+    }
+    query = (
+        'SELECT * FROM {} WHERE username=?;'
+    )
+    prepared_statement = session.prepare(
+        query.format(table)
+    )
+    try:
+        rows = session.execute(
+            prepared_statement,
+            parameters
+        )
+        if not rows:
+            raise PlantalyticsAuthException(USER_INVALID)
+        old_row = {
+            'username': rows[0].username,
+            'password': rows[0].password,
+        }
+
+        edit_row = {
+            'username': rows[0].username,
+            'password': rows[0].password,
+            'admin': rows[0].admin,
+            'email': rows[0].email,
+            'enable': rows[0].enable,
+            'securitytoken': rows[0].securitytoken,
+            'subenddate': rows[0].subenddate,
+            'userid': rows[0].userid,
+            'vineyards': rows[0].vineyards,
+        }
+        if (user_edit_info.get('username', '') != ''):
```

Comment:
Is there a way to do something like:
for each key in user_edit_info: 
    if user_edit_info.get(key, '') != ''):
        edit_row[key] = user_edit_info.get(key, '')